src/CheckoutSdk/Common/PaymentSourceType.cs

```diff
@@ -24,5 +21,14 @@ public enum PaymentSourceType
         [EnumMember(Value = "rapipago")] RapiPago,
         [EnumMember(Value = "klarna")] Klarna,
         [EnumMember(Value = "sofort")] Sofort,
+        [EnumMember(Value = "Knet")] Knet,
+        [EnumMember(Value = "qpay")] QPay,
+        [EnumMember(Value = "alipay")] Alipay,
+        [EnumMember(Value = "paypal")] PayPal,
+        [EnumMember(Value = "multibanco")] Multibanco,
+        [EnumMember(Value = "eps")] EPS,
+        [EnumMember(Value = "poli")] Poli,
+        [EnumMember(Value = "przelewy24")] Przelewy24,
+        [EnumMember(Value = "sepa")] Sepa,
```

Comment:
isn't SEPA `Value = "id"`?

Reply:
I didn't mande a full validation here, but let's make sure to add all payments in api-reference
https://github.com/checkout/checkout-api-reference/blob/master/abc_spec/components/schemas/Payments/PaymentRequestSource.yaml